test/fixtures/Todo.tsx

```diff
@@ -70,6 +70,7 @@ export const Todo = () => (
                   )}
                   <button
                     onClick={() => {
+                      console.log('add');
```

Comment:
I think you can remove this. It kinda pollutes the test output now.

Reply:
sorry for no checking the code, i will remove it later